index.d.ts

```diff
@@ -2975,6 +3004,35 @@ declare namespace Eris {
     acknowledge(choices: ApplicationCommandOptionChoice[]): Promise<void>;
     result(choices: ApplicationCommandOptionChoice[]): Promise<void>;
   }
+
+  interface ModalSubmitInteractionDataComponents {
+    components: (Pick<TextInput, "custom_id" | "type"> & { value: string })[];
+    type: Constants["ComponentTypes"]["ACTION_ROW"];
+  }
+  interface ModalSubmitInteractionData {
+    custom_id: string;
+    components: ModalSubmitInteractionDataComponents[];
+  }
+
+  export class ModalSubmitInteraction<T extends PossiblyUncachedTextable = TextableChannel> extends Interaction {
```

Comment:
alphabetical order

Reply:
The order is completely messed if we're considering alphabetical order. AutocompleteInteraction is after ComponentInteraction. Do you want me to re-arrange them?